README.md

```diff
@@ -100,253 +100,304 @@ Hurray! You successfully made a contribution!
-- [Satyam Gadekar](https://github.com/07Satyam)
-- [Saurav Kumar](https://github.com/Saurav1-2)
-- [Savio Martin](https://github.com/saviomartin)
-- [Sayed Mohammed Nouman](https://github.com/Skyhero-admin)
-- [schmelto](https://github.com/schmelto)
-- [Shanmuganathanb](https://github.com/Shanmuganathanb)
-- [Shatakshi Gupta](https://github.com/Shatakshi0805)
-- [Shaun Fitzgibbon](https://github.com/Sfitzg)
-- [Shubham Ayush](https://github.com/shubhamayush12)
-- [Shubham Hadake](https://github.com/Shubham714)
-- [Siddhant Khare](https://github.com/Siddhant-K-code)
-- [Silas Omurunga](https://github.com/Simbadeveloper)
-- [Simran Biswas](https://github.com/simranbiswas)
-- [SMak06](https://github.com/SMak06)
-- [Sohan Shetty](https://github.com/sohan9819)
-- [Sonu Kumar Kushwaha](https://github.com/flyingsonu122)
-- [Soumava Das](https://github.com/Soumava-221B)
-- [Soumya Sharma](https://github.com/soumyaa1804)
-- [Soundarya K](https://github.com/sounsrk)
-- [Srihari S](https://github.com/Liri-s29)
-- [Srinath](https://github.com/srinath1412001)
-- [Stephen Mount](https://github.com/stemount)
-- [Subhangini](https://github.com/Subhangini)
-- [Suhaas](https://github.com/suhaaskataria)
-- [Sumeet Naik](https://github.com/sumeetweb)
-- [Sumit Gaur](https://github.com/Sumit3301)
-- [Sunrit Jana](https://github.com/janaSunrise)
-- [Swapna Devi](https://github.com/Swapna804)
-- [syedareehaquasar](https://github.com/syedareehaquasar)
-- [Sefat Siddiquea Sifa](https://github.com/sifa123)
-- [Shyam Sutar](https://github.com/shyamsutar)
-- [Taneem Jan](https://github.com/taneemishere)
-- [Tanya Powell](https://github.com/tanyapowell)
-- [Tawfik Yasser](https://github.com/TawfikYasser)
-- [Tomo Bones](https://github.com/tomobones)
-- [Triman Kaur](https://github.com/Trimankaur)
-- [Tawfik Yasser](https://github.com/TawfikYasser)
-- [Udokaku Ugochukwu](https://github.com/UdokaVrede)
-- [Ujjwal Gupta](https://github.com/UG-SEP)
-- [Umesh Singh](https://github.com/Umesh-01)
-- [Usman Tatla](https://github.com/iTatla1)
-- [Utkarsh Singh](https://github.com/Utkarsh299-tech/)
-- [Utkarsh Mishra](https://github.com/Utkarsh1504)
-- [Vamsi Ravi](https://github.com/vamsi963601)
-- [Varghese Jose](https://github.com/varghesejose2020/)
-- [Vaibhav Wani](https://github.com/Vaibhavwani11)
-- [Venkata Mouli](https://github.com/mouli3029)
-- [Vicky Gupta](https://github.com/viki4545)
-- [Victory Chiamaka Wekwa](https://github.com/VictoryWekwa)
-- [Vikas Ganiga](https://github.com/vikasganiga05/)
-- [Vince Y](https://github.com/vmyu)
-- [Vinicius Ramos](https://github.com/redrookie)
-- [Vittorio Rinaldo](https://github.com/rinaldovittorio)
-- [Vivek Singh](https://github.com/vivmost)
-- [Vladyslav Kyiashko](https://github.com/vlakyi)
-- [Yashasvi Misra](https://github.com/yashasvimisra2798)
-- [Yaswanth Modepalli](https://github.com/Yaswanth14)
-- [Yuvraj Singh Chouhan](https://github.com/Yuvrajhere)
-- [Zoheb Alli Khan](https://github.com/ZohebMOPO)
+* ### **A**
```

Comment:
Don't know if the bold is necessary on the headings. But I leave this open for discussion.

Reply:
> 
> 
> Don't know if the bold is necessary on the headings. But I leave this open for discussion.

I think it will look good